homeassistant/components/intellifire/strings.json

```diff
@@ -1,39 +1,30 @@
 {
   "config": {
-    "flow_title": "{serial} ({host})",
+    "flow_title": "{serial}",
     "step": {
-      "manual_device_entry": {
-        "description": "Local Configuration",
-        "data": {
-          "host": "Host (IP Address)"
-        }
-      },
-      "api_config": {
+      "pick_cloud_device": {
+        "title": "Configure fireplace",
+        "description": "Select Fireplace by Serial Number:"
+      },
+      "cloud_api": {
+        "description": "Authenticate against IntelliFire Cloud",
+        "data_description": {
+          "username": "IntelliFire Username",
+          "password": "IntelliFire Password"
```

Comment:
This can be a bit more descriptive, this now sounds like it's a header. Something along the lines of "Your IntelliFire cloud username" or something like that